packages/tokens/breadcrumb.ts

```diff
@@ -0,0 +1,5 @@
+import type { InjectionKey } from 'vue'
+import type { BreadcrumbProps } from '@element-plus/components/breadcrumb/src/breadcrumb'
```

Comment:
Can we use 

```ts
import type { BreadcrumbProps } from '@element-plus/components/breadcrumb'
```
here?

Reply:
Even though for types it won't raise error but we should be carefully handling these imports.

Reply:
If do this, the `hooks` build will failed.